hoss/subset.py

```diff
@@ -122,16 +124,26 @@ def subset_granule(
             shape_file_path = get_request_shape_file(
                 harmony_message, output_dir, logger, config
             )
+            logger.info(
+            'All required variables: ' 
+            f'{format_variable_set_string(required_variables)}' 
+            ', dimensions_path: ' f'{dimensions_path}' 
+            ', shapefilepath:' f'{shape_file_path}'
+            )
```

Comment:
This code is improperly indented. The contents of the parentheses should be indented one more time. I'm surprised the `pre-commit` hooks didn't catch this.

Reply:
I don't see this indentation....
That link to subset.py seems to be a link to an outdated version?

Reply:
That's weird! It's gone now though, so never mind this comment. :disappear: